readthedocs/settings/base.py

```diff
@@ -240,6 +243,19 @@ def USE_PROMOS(self):  # noqa
     CELERY_CREATE_MISSING_QUEUES = True
 
     CELERY_DEFAULT_QUEUE = 'celery'
+    CELERYBEAT_SCHEDULE = {
```

Comment:
We need to make sure we aren't overridding this elsewhere. We need a good pattern here, but I'm guessing we aren't extending this everywhere else we use it, so this might get killed by a downstream config file overwriting it.

Reply:
I found we use this setting in 3 places:

* pydoc.io, but I think this project doesn't extend from RTD.org, so nothing to worry about
* org operations, which I've already made a PR to use the `@property`
* corporate site (https://github.com/readthedocs/readthedocs-corporate/blob/master/readthedocsinc/settings/base.py#L175) which I suppose that I need to do the same than I did for org.

@ericholscher please confirm this, and I will open a PR for corporate ops to modify this file https://github.com/readthedocs/readthedocs-corporate-ops/blob/master/salt/corporate/readthedocsinc/settings/base.py

Reply:
You need a property override there as well, correct.